### PR TITLE
feat/19 prepare v1

### PR DESCRIPTION
**Pull request by malinovic**
Hi, this merge request contains following updates:

- remove node 10 support
- update dev dependencies to a newer version

With this merge request I also updated the version to the v1.0.0. This merge request closes following issue: https://github.com/merkle-open/env-linter/issues/19

If there are any open questions, please get back to me as soon as possible :)

Best regards,
Danijel

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,17 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.0.0](https://github.com/merkle-open/env-linter/compare/v0.5.1...v0.6.0) (2022-04-29)
```
Comment (ernscht) [inline CHANGELOG.md:5]: compare v0.5.1 with v1.0.0

**Review by ernscht — CHANGES_REQUESTED** [review]
There is nothing else to say against a release 1.0.
Of course there are again some dev dependencies that could be renewed, as well as the default node version, but that can always be done.

You could still run prettier (results in adjustments in the changelog)

**Comment by ernscht** [comment]
> The only thing I noticed (which is an issue on the develop-branch as well) is that `npm run lint` exits with an error. This could be fixed by adjusting the package json: `"lint:ts": "eslint . --ext .jsx,.js,.ts,.tsx",` to `"lint:ts": "eslint --ext .jsx,.js,.ts,.tsx",`

For me, the linting does not give any errors. It is only a warning.
And changing the "lint:ts" task like this would mean that nothing is linted anymore and this certainly does not give any errors anymore ;-)

File: CHANGELOG.md (hunk 1 above)
Comment (malinovic) [reply CHANGELOG.md:5]: > There is nothing else to say against a release 1.0. Of course there are again some dev dependencies that could be renewed, as well as the default node version, but that can always be done.
> 
> You could still run prettier (results in adjustments in the changelog)

I see your point of view. These package version change quite frequently and it is sometimes hard to keep track of... Nonetheless, the default node version in the .node-version file can be updated for sure.

**Comment by malinovic** [comment]
> > The only thing I noticed (which is an issue on the develop-branch as well) is that `npm run lint` exits with an error. This could be fixed by adjusting the package json: `"lint:ts": "eslint . --ext .jsx,.js,.ts,.tsx",` to `"lint:ts": "eslint --ext .jsx,.js,.ts,.tsx",`
> 
> For me, the linting does not give any errors. It is only a warning. And changing the "lint:ts" task like this would mean that nothing is linted anymore and this certainly does not give any errors anymore ;-)

I noticed the warning and I have the same behaviour as @ernscht. I guess as long as it's a warning it can be *ignored*. For a future release though it would be better to remove this warning in the best way possible. I will create a new issue in the next time.

**Comment by ernscht** [comment]
> I will create a new issue in the next time.
the warning comes from the eslint-config – so, no chance to fix it here ;-)

**Comment by malinovic** [comment]
> > I will create a new issue in the next time.
> 
> the warning comes from the eslint-config – so, no chance to fix it here ;-)

Ahhh I see - nevermind, ignore my comment then 😄

**Review by ernscht — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -18,52 +18,52 @@
...
+    "tslib": "2.4.0",
+    "typescript": "4.6.4",
     "webpack": "4.46.0",
-    "webpack-cli": "4.8.0",
+    "webpack-cli": "4.9.2",
     "zen-observable": "0.8.15"
   },
   "engines": {
-    "node": ">=10.15.3"
+    "node": ">=v14.15.0"
```
Comment (ernscht) [inline package.json:66]: I think "v" is not valid here

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,13 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.0.0](https://github.com/merkle-open/env-linter/compare/v0.5.1...v1.0.0) (2022-04-29)
```
Comment (ernscht) [inline CHANGELOG.md:5]: should we change the date?

**Review by ernscht — CHANGES_REQUESTED** [review]

**Review by ernscht — APPROVED** [review]

**Pull request merged**